src/prefect/agent/kubernetes/agent.py

```diff
@@ -10,6 +12,20 @@
 from prefect.serialization.storage import StorageSchema
 from prefect.utilities.graphql import GraphQLResult
 
+AGENT_DIRECTORY = path.expanduser("~/.prefect/agent")
```

Comment:
At first I thought we needed to change this for windows compatibility but this is intended to run inside a Linux docker container so we're good!

Reply:
Yes sir!